app/src/main/java/com/harman/wearosapp/app/ui/feature_screen/FeatureListFragment.kt

```diff
@@ -0,0 +1,94 @@
+package com.harman.wearosapp.app.ui.feature_screen
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.AppCompatButton
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearSnapHelper
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.SnapHelper
+import androidx.wear.widget.WearableLinearLayoutManager
+import com.harman.wearosapp.app.R
+import com.harman.wearosapp.app.adapter.FeatureListAdapter
+import com.harman.wearosapp.app.databinding.FragmentAppFeatureListBinding
+import com.harman.wearosapp.domain.model.FeatureModel
+import kotlin.math.abs
+
+
+class FeatureListFragment : Fragment() {
+
+    private lateinit var binding: FragmentAppFeatureListBinding
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentAppFeatureListBinding.inflate(inflater, container, false)
+
+        val lnManagerCallBack = object : WearableLinearLayoutManager.LayoutCallback() {
+            var relativeToCenter: Float = 0f
+            var progressToCenter: Float = 0f
+            override fun onLayoutFinished(child: View?, parent: RecyclerView) {
+                child?.apply {
+                    val offset = height.toFloat() / 2.0f / parent.height.toFloat()
+                    relativeToCenter = y / parent.height + offset
+                    progressToCenter = abs(0.5f - relativeToCenter)
+                    val button = findViewById<AppCompatButton>(R.id.btFeatureButton)
+                    button.background = if (progressToCenter < 0.15f) ContextCompat.getDrawable(
```

Comment:
lets just change color transparency (not replace background view) based on progressToCenter

Reply:
done